ansible_collections/arista/avd/roles/eos_designs/python_modules/base/avdstructuredconfig.py

```diff
@@ -41,6 +43,11 @@ def router_bgp(self) -> dict | None:
             "as": self.shared_utils.bgp_as,
             "router_id": self.shared_utils.router_id,
             "bgp_defaults": bgp_defaults,
+            "bgp": {
+                "default": {
+                    "ipv4_unicast": bgp_default_ipv4_unicast,
```

Comment:
and if. you do this you can remove line 34/35
```suggestion
                    "ipv4_unicast": get(self._hostvars, "bgp_default_ipv4_unicast", default=False),
```